Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -416,10 +406,14 @@ def bottle_formula(f, args:)
         cd cellar do
           sudo_purge
           # Tar then gzip for reproducible bottles.
-          tar, tar_args = setup_tar_and_args!
-          # Use portable tar format to match libarchive format.
-          safe_system tar, "--create", "--numeric-owner", "--format", "ustar",
-                      *tar_args,
+          tar = setup_tar!
+          # Ensure tar is set up for reproducibility.
+          # https://reproducible-builds.org/docs/archives/
+          safe_system tar, "--create", "--numeric-owner", "--format", "pax",
+                      "--owner", "0", "--group", "0", "--sort", "name",
```

Comment:
Can't rely on `--owner` and `--group` here as this may not be `gnu-tar` if the formula isn't present. I'd suggest continuing to return arguments from `setup_tar_and_args!`.